pages/en/lb4/Creating-decorators.md

```diff
@@ -0,0 +1,13 @@
+---
+lang: en
+title: 'Creating decorators'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Creating-decorators.html
+summary: A tutorial to use `@loopback/metadata` module to create new decorators
+---
+
+[Create decorators](https://github.com/strongloop/loopback-next/blob/master/packages/metadata/README.md)
```

Comment:
An empty page with a single link to external README is suboptimal. Could you please use the mechanism we have already in place for sourcing content from README files?

See http://loopback.io/doc/en/contrib/Including-READMEs.html and http://loopback.io/doc/en/contrib/readmes.html

Reply:
Yes, I was trying to do so but the current tooling does not seem to support a monorepo (loopback-next).

Reply:
> the current tooling does not seem to support a monorepo (loopback-next).

Could you please be more specific? What is the current problem and how much work would it be to fix it? If you prefer to land this PR first, then I am ok with that as long as there is a follow-up issue so that we don't forget to fix this problem.

Reply:
Is this discussion actually relevant, considering 7e95ae0 and 934c889?